components/board/jetson/data.go

```diff
@@ -94,8 +94,8 @@ var jetsonNanoPins = []genericlinux.PinDefinition{
 	{Name: "31", DeviceName: "gpiochip0", LineNumber: 200, PwmChipSysfsDir: "", PwmID: -1},
 	// Older versions of L4T have a DT bug which instantiates a bogus device
 	// which prevents this library from using this PWM channel.
-	{Name: "32", DeviceName: "gpiochip0", LineNumber: 168, PwmChipSysfsDir: "7000a000.pwm", PwmID: 0},
-	{Name: "33", DeviceName: "gpiochip0", LineNumber: 38, PwmChipSysfsDir: "7000a000.pwm", PwmID: 2},
+	{Name: "32", DeviceName: "gpiochip0", LineNumber: 168, PwmChipSysfsDir: "", PwmID: -1}, // "7000a000.pwm", PwmID: 0},
+	{Name: "33", DeviceName: "gpiochip0", LineNumber: 38, PwmChipSysfsDir: "", PwmID: -1},  // "7000a000.pwm", PwmID: 2},
```

Comment:
I'm not at all familiar with this type of data file, but I notice that all the other PwmIDs changed from 0 to -1. why does this one change from 2 to -1?

Reply:
-1 means "this pin has no hardware PWM associated with it," so removing HW PWM is done by setting them all to -1.

If the PWM ID is _not_ -1, it's the line on the PWM chip that connects to this pin. Most chips NVidia uses have 1 line per chip, and they add extra chips to get extra PWM capabilities. However, some boards use a PWM chip with 4 lines coming out of it: line 0 is attached to physical pin 32, line 1 IIRC is the cooling fan on the CPU's heatsink, line 2 is physical pin 33, and line 3 is... something else I don't remember.